src/Dimensions/dimension.jl

```diff
@@ -303,22 +303,25 @@ Base.size(dims::DimTuple) = map(length, dims)
 Base.CartesianIndices(dims::DimTuple) = CartesianIndices(map(d -> axes(d, 1), dims))
 
 # Extents.jl
+#=
```

Comment:
Why was this commented out?

Reply:
So that we still had the old version to refer back to, mainly

Reply:
can just remove it, but we probably need an explanation for the new complexity

Reply:
Yea, now that it works I'll do that